pkg/generator/steps/layout/tpl-new/py-services/_service_/generated/app/server.py.tpl

```diff
@@ -0,0 +1,77 @@
+{{.TplHeader}}
+
+import asyncio
+
+from aiohttp import web
+
+from libraries.generated.metrics.server import MetricsServer
+
+from {{.ServiceName}}.generated.openapi import app
+from {{.ServiceName}}.generated.core.service_context import MifyServiceContext
+
+
+class ServerConf:
+    API_ENDPOINT: str = '{{.ApiEndpoint}}'
+    MAINTENANCE_ENDPOINT: str = '{{.MaintenanceEndpoint}}'
+
+    _env_mapping: dict = {
+        'API_ENDPOINT': '{{.ApiEndpointEnv}}',
+        'MAINTENANCE_ENDPOINT': '{{.MaintenanceEndpointEnv}}',
```

Comment:
Let's choose one style for python strings and use it. Now in some parts, you are using ' and in others ". My vote is for " everywhere